dts/common/nordic/nrf54l05.dtsi

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nordic/nrf54l_05_10_15.dtsi>
+
+/* 72 + 24 = 96KB */
+&cpuapp_sram {
+	reg = <0x20000000 DT_SIZE_K(72)>;
+	ranges = <0x0 0x20000000 DT_SIZE_K(72)>;
+};
+
+&cpuflpr_sram {
+	reg = <0x2002f000 DT_SIZE_K(24)>;
```

Comment:
This is incorrect. The start address should be 0x20012000, right after `cpuapp_sram` (0x2002f000 is out of range of the available SRAM).
I think the memory nodes, at least the FLPR ones, should not be defined in the common file but rather in the SoC specific files (because the start addresses are part of their names).